src/checkpoint/checkpoint_txn.c

```diff
@@ -2714,8 +2721,12 @@ __wt_checkpoint_close(WT_SESSION_IMPL *session, bool final)
     if (final && !metadata)
         return (__wt_evict_file(session, WT_SYNC_DISCARD));
 
-    /* Closing an unmodified file. */
-    if (!btree->modified && !bulk)
+    /*
+     * Evict the unmodified file without taking a checkpoint when closing the handle. If we're in
```

Comment:
to ensure we write the bitmap (tracking the migrated data) to the files metadata entry. -> to ensure we write the live restore metadata.

Reply:
https://github.com/wiredtiger/wiredtiger/pull/11725/commits/93b128178f634cfddfa51fd636a31e8c4e18ad7b